sdk/formrecognizer/Azure.AI.FormRecognizer/MigrationGuide.md

```diff
@@ -569,7 +569,9 @@ Differences between the versions:
 Train a custom model with `3.1.x`:
 ```C# Snippet:FormRecognizerSampleTrainModelWithFormsAndLabels
 // For this sample, you can use the training forms found in the `trainingFiles` folder.
-// Upload the forms to your storage container and then generate a container SAS URL.
+// Upload the forms to your storage container and then generate a container SAS URL. Note
+// that a container URI without SAS is accepted only when the container is public or has a
+// managed identity configured.
```

Comment:
```suggestion
// managed identity configured.
//
```

Since you're starting a new paragraph, I think the line break helps with readability.

Reply:
This is part of an auto-generated snippet. I'll modify the original code and regenerate.